compute/compute-node.Dockerfile

```diff
@@ -1475,6 +1475,8 @@ RUN mkdir -p /etc/local_proxy && chown postgres:postgres /etc/local_proxy
 COPY --from=postgres-exporter /bin/postgres_exporter /bin/postgres_exporter
 COPY --from=sql-exporter      /bin/sql_exporter      /bin/sql_exporter
 
+COPY --chown=postgres compute/etc/postgres_exporter.yml /etc/postgres_exporter.yml
```

Comment:
So it wasn't working, and nobody noticed? Do we use metrics from `postgres_exporter` anywhere?

Reply:
No, it works. It just logs an ERROR. I'm not sure if we are using any stats from postgres-exporter. It's something we should check.